lib/theme_check/checks/template_length.rb

```diff
@@ -3,6 +3,7 @@ module ThemeCheck
   class TemplateLength < LiquidCheck
     severity :suggestion
     category :liquid
+    doc docs_url("docs/checks/template_length.md")
```

Comment:
Since this will only be used in checks, should the `docs/checks/` prefix be moved to the helper method?

Reply:
But at the same time that's very magicky. Doesn't make it clear where the docs are really. Whereas with the full path here you can pretty much copy paste a Check without context and understand where the docs are supposed to be.